### ISSUE TITLE
Moderation of tests by teammembers should be applied

### ISSUE THREAD

**Issue by semeyerz** [RoboCupAtHome/RuleBook#422]
3.8.13 is a whole section on moderation which I would love to see in action.
Maybe there should be either points or penalties applied to foster this.

**Comment by warp1337** [comment]
+1

**Comment by kyordhel** [comment]
No bonus points. No team should score for helping in logistics. In fact, somewhere in the rulebook states teams must provide a person for moderation, and not showing up (for moderation) has a penalty already established of 50 points (same as not showing up for the test).

OC has not enough people to moderate, nor to chase teams and apply penalties. We can, however, be more strict and penalize al teams whose tests weren't moderated. However, as OC Chair I strongly stand against this, otherwise we will have tons of negative scoring.

**Comment by warp1337** [comment]
This is a 'double-edged sword'. On the one hand, I approve @semeyerz opinion that a moderated task will provide the audience with a better understanding of what is currently happening in the arena. This has been discussed a lot during the last years. On the other hand, it is extremely difficult to "assess" moderation. As an example: let's assume one of the team members actually shows up to moderate... is "the robot enters the arena now", "the robot tries to reach it navigation goal", ... some time passes and the robot does all it's tasks, "the robot leaves the arena" considered moderation? To get to the point: if we award moderation, there should be a metric applied otherwise this will lead to lengthy discussions of what "moderation" is.

**Comment by LoyVanBeek** [comment]
+1 for some form of moderation/explanation. 

On competitions like the European Open, there was a non-team moderator for some tests. Knowing when to shut up and give the robots a chance to understand the operator is non-trivial.
Also, not everyone is comfortable addressing a crowd... 

The vizbox should help to give the audience an idea of what is going on, but cannot do in-depth explanation of why a task is hard etc. 
Usage of the vizbox is also not enforced of course.

**Comment by semeyerz** [comment]
I suggest linking the moderation points to the points of the poster session.

**Comment by warp1337** [comment]
Still, the questions how do you measure "moderation"?

**Comment by kyordhel** [comment]
I would like to remark here that RoboCup@Home is about testing and benchmarking robots. Therefore, anything else but a robot performance shouldn't be scored. TC can't simply place bonuses and penalties at will. In fact, such changes must be discussed first with the Execs.

The only global penalty in the Rulebook is the one for not showing up, and it should be kept this way (reasons are way too obvious to be detailed here). In that sense, if we are strict, the TC/OC can require any team to provide a person to perform as volunteer, referee, or moderator. If that person doesn't show up, the team is penalized he same way as if it hasn't attended to a test.

Such policy has been used in the past, but forcing people to take part as referee/moderator causes more harm than helps.
- As referee...
  - One can directly cheat while scoring other teams
- As moderator...
  - One spoil a test by speaking during speech-recognition tasks
  - One may not know the test, misinforming the audience
  - One may have stagefright
  - One may not moderate at all, concerned of what is going wrong with the robot

In practice, the OC would need to provide supervisors to check what moderators are doing right. Unfortunately, only in rare occasions the OC has enough people to fulfill all its duties, so this is simply not doable. In the past, several times we had to bend the rules because the professional moderator provided by the venue made impossible to perform ASR.

In conclusion, rules are fine, but the league lacks the profile and commitment to provide a decent moderation.

**Comment by semeyerz** [comment]
I think this Issue goes pretty well with PR #438 as this PR encourages teams to watch the performance of other teams as this is the only point to complain.

Also it would go quite well with the poster session as teams rate each others "moderation" performance in this test.

Why not just add another score/column in the table for moderation of all tasks to the points of poster session?
The only disadvantage I see right now is that the scores for Poster session could only be released after all tests are done but I don't see a big problem with this.

**Comment by kyordhel** [comment]
@balkce @dirkholz @dragonjack @swachsmu, EC is required here.
@robotijn, @komeisugiura, @iocchi, You might also want to comment since it involves scoring.

**Comment by balkce** [comment]
Let me start wearing my "negative" hat, to get the bad part out of the way first.
In terms of points, making teams judge themselves moderation-wise is a hard sell:
- Adding this score to the Poster Session implies that these scores will need to be provided at the end of Stage II since these tests should also be moderated.
- The idea behind #438 deals with a different matter: score protests can be made more robust by having all the teams be present to have them discuss, and it's not mandatory. However, adding a moderation score will deviate from that since they would ALSO need to observe how well the moderation is going.
- This score will give an unfair advantage to teams that come from countries with a high level of English or those that bare a thick accent.
- Team members are seldom available because of their understood interest of being backstage making sure the robot is ready before a test. Making teams give up on one member to volunteer for one test (Speech and Person Recognition) is difficult enough, I can't imagine how difficult it would be to convince them to give up one member for all the tests.

Before I take off my "negative" hat, it's important to remember that moderation benefits the audience, and we should we keep that separate from scores, which benefits the teams. We have had very bad experiences with mixing crowd-pleasing stunts with scoring, and I'm not convinced that we should do that again.

Now, I'll take off my "negative" hat, and offer an olive branch. I think we all agree that having moderation is a good thing (public interest brings good morale to the team as well as sponsors). The problem with moderation in RoboCup@Home is how to carry it out, and with all due respect to the teams, I don't trust them to carry this burden if it's optional.

So, I think we should not half-ass it and push to have an official moderator for each league (OPL, SSPL, DSPL). This should be a person that knows the tests (so he/she knows when to shut up) and is good talking to crowds. EC ( @dirkholz @swachsmu @dragonjack ), I'm looking at you. I'm game if need be. Another alternative is that we can push to have an additional TC member per league for next year just for this task.

I would also add a very important part to this: team members can opt in to complement the official moderation. This is very similar to what is done in the drone competition during IROS, where there is an official moderator that is talking to the audience and a member of the team talks along with him. The beauty of this is that if a team opts out from this, their "penalty" comes from the audience response, not from their score, keeping them separate. Basically, if no team member moderates, the drone may not be as well received as others by the audience since there was nobody hyping it from the team; that's it. Yes, teams probably won't participate anyway and the official moderator will probably end up doing his/her thing alone, but the ones that do participate will probably be very popular with the audience.

**Comment by awesomebytes** [comment]
I was moderating on the Rescue league for my team some years ago (dunno if
it was mandatory, I was asked to do it and I just did it) and I must say
it's not that much work (if you do it for your own team test) and the
audience gets way more involved. I feel unfair adding scoring to it but I
don't think it's asking too much to explain a bit of what's going on. I
prefer a boring "it's trying to navigate to the room" than nothing. I think
we all have explained at some point to spectators what's going on in the
tests just cause they were wondering openly just next to us, haven't we?

I offer myself to do it in Montreal for my team maybe to just give example.

**Comment by LoyVanBeek** [comment]
> I believe at the German Open we had that at some point, i.e., Prof. Paul Ploeger and also @swachsmu did the moderation.

From my experience, when they did do some moderation (in the native language of the audience) there usually was a larger crowd or it gathered one if there was none.

**Comment by komeisugiura** [comment]
From RCF's perspective, @home competitions should be more easily understood by audience. 
Here is a hint:
Why teams are willing to "moderate" in the Finals without forced to do?

**Comment by balkce** [comment]
> @home competitions should be more easily understood by audience

I don't think having a moderator and having an easily understood test are mutually exclusive. Football is widely well understood, but people still moderate it: it's like having a narrator in a movie. I agree with @LoyVanBeek, when there has been moderation, a group of people always gathered around the arena.

> Why teams are willing to "moderate" in the Finals without forced to do?

Because they have a panel of judges that they're trying to persuade for points. In fact, in Mexico 2012, during the Open Challenge, I was interrupted by an EC member because "I was presenting to the audience, not the judges."

Granted, if I ever saw a demo in the Finals in which the robot did everything without requiring a team member to moderate and I understood it perfectly, I'd be really impressed.


But we're getting off-track here: I believe we should have an official moderator (from the EC/TC/OC). Unfortunately, there's the issue of not having enough people. We barely have enough to score, order the queue, take time, place objects, etc. I think I would've done it during Nagoya for DPSL if I didn't need to keep score. Thus, we need people just for this task.

@awesomebytes has offered his services for his team (and MAYBE can be convinced to do it for others in SSPL *squint* *squint*)

I'm up for doing for either DPSL or OPL.

Who else?

If this seems like something we can't work out before Montreal, we can leave it at that, but I would definitely consider it for the next TC.

**Comment by kyordhel** [comment]
@balkce I wouldn't modify the rulebook at all for this, and can't organize anything now, but I'm open to **ANYONE** who wants to officially moderate in any of the arenas (I'm OC Chair after all), or help the organization in any other way. The same as in Nagoya, but you know, we lack of proficient English speakers.

**Comment by kyordhel** [comment]
@semeyerz are you suggesting we score people and not only robots in RoboCup@Home?

**Comment by semeyerz** [comment]
> @semeyerz are you suggesting we score people and not only robots in RoboCup@Home?

exactly! Same as in open challenge, poster session and finals.
In the end we always score teams. Behind the robots is always a group of humans.

At least this is how I interpret the comment from Komei.

**Comment by balkce** [comment]
> we should think of a way to give points in a fair and easy way

I can't think of a scoring mechanism for this that can't be abused in a way that a robot that doesn't do anything and should not received any points but will be awarded points anyway because a team member can just say into a microphone "Hello, I'm from Team A".

We already had a long discussion in #425 about teams hogging the arena with inoperative robots in the hope of some points. If not done right, this will open the door for more of this abuse.

Nonetheless, this could be a limitation in my intellect. So, sure, show me a mechanism that can't be abused like that, and I'll vote for it.

**Comment by semeyerz** [comment]
At first someone needs to decide whether we really want to score points for moderation or not.

If yes one idea could be that teams mutually vote whether the other teams did good moderation.
Like good +1, okish 0, no/bad -1
And then it is applied the same way as data recording was.
Meaning it is dependent on the score of the robot in the task.
Could be like 10% x score x moderation + score.
So if a robot scores 50 it will be 55 if there was good moderation or 45 if there was none.

Or it could be even done per stage so that the oc only needs to collect points one time during the day for moderation.

But I would think more about that when it is decided that it is really wanted.

**Comment by kyordhel** [comment]
I'm directly asking @balkce @dirkholz @dragonjack @swachsmu as Executive Committee:

Should teams receive points for moderating tests?

If so, how?

**Comment by balkce** [comment]
> Should teams receive points for moderating tests?

I don't believe they should.

**Comment by justinhart** [comment]
Shouldn't this, on some level, be fun? Why can't teams moderate tests just
because they like being at RoboCup and it's another way to participate? I
don't want points for having a beer and watching the sunset.

**Comment by swachsmu** [comment]
There should be no points for moderating tests. Team should do it
because they want to promote the @home community and their team to the
public.
For Montreal, it would be ideal to have professional moderators who can
speak English as well as French. This is an issue for the local OC.
Team moderators could join the professional moderators, answer question
and give comments.

Best regards
Sven


Am 16.05.2018 um 21:41 schrieb Mauricio Matamoros:

**Comment by LoyVanBeek** [comment]
> Team should do it because they want to promote the @home community and their team to the public.

I think this is important here. I think teams are somewhat sensitive to this rhetoric if brought well. 

I'll volunteer to do moderation on 1 or 2 challenges for OPL BTW.

**Comment by kyordhel** [comment]
I think this settles the issue. I'll review pertinent sections and update them accordingly.
